client/src/controllers/scores/scores_controller.js

```diff
@@ -102,6 +109,27 @@ class ScoresController {
 		return badScores;
 	}
 
+	teamIsSelected() {
+		return !this._loading && this._teamNumberList && this._teamNumberList.indexOf(parseInt(this.search)) > -1

```

Comment:
What is the meaning behind `this._teamNumberList.indexOf(parseInt(this.search)) > -1`? What is the logical connection between the question of wether the team is selected to the search mechanism of the scores. And why is this all in the `scores` and not in the `score`?